boldpredict/forms.py

```diff
@@ -23,10 +23,12 @@ def clean(self):
 
         username = cleaned_data.get('username')
         password = cleaned_data.get('password')
-        user = authenticate(username = username, password = password)
-        if not user:
+        user1 = authenticate(username = username, password = password)
+        user=User.objects.get(username=username)
```

Comment:
If the user doesn't exist, it will create an error, Use this:
        try:
               user = User.objects.get(username=username)
        except User.DoesNotExist:
            return ValidationError('User does not exist')